CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Ewon Flexy Extensions Library Changelog
 
-## Version 1.13.5
+## Version 1.13.6
```

Comment:
Was skipping 1.13.5 intentional?

Reply:
Good catch. 1.13.5 was not skipped, although it did fail to deploy to the Maven repo. 

This should've been a separate changelog for 1.13.6.

Reply:
fixed